omod/src/main/webapp/fragments/field/datetimepicker.gsp

```diff
@@ -39,14 +39,9 @@
     def defaultDateString = ""
     def defaultDateISOFormatted = ""
     if (defaultDate) {
-        if( ui.convertTimezones() ) {
-            if(useTime){
+        if( ui.convertTimezones() && useTime) {
                 defaultDateString = ui.format(defaultDate)
-                defaultDateISOFormatted = ui.format(defaultDate)
-            } else {
-                defaultDateString = ui.formatDateWithClientTimezone(defaultDate)
```

Comment:
Did you intentionally remove the use of `formatDateWithClientTimezone()`?

Reply:
Yes, that was intentional, we notice that if we don't have time associated with a date, timezones don't make sense because we only have a date, that is why we remove that.

Reply:
How about when `useTime` is true but `convertTimezones` is false?

Reply:
We only make conversions if they are both true `useTime ` and  `convertTimezones `:
https://github.com/openmrs/openmrs-module-uicommons/pull/92/files/8af0551e5822501d87b0686679e0402a35830f8d#diff-5bd2e80e4ad99f1d821cef9905045f3f1e185334264bb82f7e81ced1540acc55R42